pallets/ajuna-awesome-avatars/src/tests.rs

```diff
@@ -574,15 +581,20 @@ mod minting {
 
 		let mint_cooldown = 5;
 
+		let initial_balance = 1_234_567_890_123_456u64;
+
 		ExtBuilder::default()
 			.organizer(ALICE)
 			.seasons(vec![season.clone()])
 			.mint_availability(true)
 			.mint_cooldown(mint_cooldown)
+			.balances(vec![(ALICE, initial_balance)])
```

Comment:
We need `mint_fees` method for `ExtBuilder` so we can test fees

Reply:
Added